.github/workflows/ci.yml

```diff
@@ -101,35 +104,36 @@ jobs:
       - name: Run linters
         run: poetry run invoke lint
 
-
-  deploy:
-    name: Deploy
+  prep-release:
+    name: Release preparations
     environment: Deployment
     needs: [test, lint]
     runs-on: ubuntu-latest
     if: ${{ github.ref=='refs/heads/main' && github.event_name!='pull_request' }}
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Setup Python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
         with:
           python-version: "3.8"
       - name: Check release
         id: check_release
         run: |
           python -m pip install --upgrade pip
-          python -m pip install poetry githubrelease httpx==0.16.1 autopub
-          echo "##[set-output name=release;]$(autopub check)"
-      - name: Publish
+          python -m pip install poetry autopub[github]
```

Comment:
I assume the `github` extra works just fine?

Reply:
Yes, I believe the `github` extra should work fine. There was a point at which I pinned `httpx` in order to work around an issue with `githubrelease`, but I imagine that has since been resolved. I'd rather take the approach you have here and then re-pin `httpx` if and when needed. 👍